cmd/kube-proxy/app/options/options.go

```diff
@@ -65,6 +66,7 @@ func NewProxyConfig() *ProxyServerConfig {
 func (s *ProxyServerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(componentconfig.IPVar{Val: &s.BindAddress}, "bind-address", "The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces)")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig)")
+	fs.StringSliceVar(&s.APIServerList, "api-servers", []string{}, "List of Kubernetes API servers for publishing events, and reading pods and services. Comma separated list of hostnames or host:port pairs.")
```

Comment:
i added api-servers to kube-proxy so that it will be uniform with kubelet, and ofcourse there is Servers options for kubeconfig